Gruntfile.js

```diff
@@ -14,50 +14,43 @@
  * limitations under the License.
  */
 
+
 module.exports = function(grunt) {
+  const closurePackage = require('google-closure-compiler');
+  closurePackage.grunt(grunt);
 
   grunt.initConfig({
     pkg: grunt.file.readJSON('package.json'),
 
-    closureDepsWriter: {
-      options: {
-        closureLibraryPath: 'third_party/closure-library',
-      },
-      helperDeps: {
-        src: 'src/helper/helper.js',
-        dest: 'src/deps.js'
-      }
-    },
-
-    closureBuilder:  {
-      options: {
-        closureLibraryPath: 'third_party/closure-library',
-        compilerFile: 'third_party/closure-compiler/compiler.jar',
-        namespaces: 'helper',
-        compile: true,
-        compilerOpts: {
+    // Using https://www.npmjs.com/package/google-closure-compiler
+    'closure-compiler': {
+      my_target: {
+        files: {
+          'dist/data-layer-helper.js': 'src/helper/**.js',
+        },
+        options: {
+          js: [
+            'node_modules/google-closure-library/closure/goog/base.js',
+            'src/plain/**.js',
+          ],
+          hide_warnings_for: 'google-closure-library',
           compilation_level: 'ADVANCED_OPTIMIZATIONS',
+          language_in: 'ECMASCRIPT6',
+          create_source_map: 'dist/data-layer-helper.js.map',
```

Comment:
Not sure this is entirely useful. We've never had a source map before, so unless there's a good reason to add it, I'd remove this. If we decide the source map is useful, we can add back in a separate PR.

Reply:
Sure thing, I got rid of it.